css-box-model/styles.css

```diff
@@ -0,0 +1,9 @@
+button.language {
+  font-size: 24px;
+  background-color: lightgray;
+
+  /* your code here 👇 */
+  margin: 0 12px;
```

Comment:
Try it without specifying the top or bottom margins

Reply:
You got it! Will fix!

Reply:
New screenshot:
![image](https://user-images.githubusercontent.com/85596835/144292022-c2075512-6594-4e5f-b74b-3ad3f03e172a.png)